data/grand-rick001.json

```diff
@@ -2,6 +2,7 @@
   "name": "Patrick Murimi",
   "type": "personal",
   "bio": "I am a Junior Full Stack Web Developer, specializing in JavaScript | GDSC Web Lead | Postman Student Expert",
+  "avatar": "https://github.com/grand-rick001.png",
```

Comment:
Hi @grand-rick001  No need to add avatar, You can remove it

Reply:
Yeah, just wanted to add a custom one to my LinkFree profile

Reply:
> 

looks like you add your github avatar, so in recent update your github profile pic will be add automatic, so you don't need `"avatar": "https://github.com/grand-rick001.png",`. you can remove it

Reply:
Okay, on it